packages/lesswrong/lib/make_voteable.ts

```diff
@@ -133,12 +134,9 @@ export const makeVoteable = <T extends DbVoteableType>(collection: CollectionBas
     baseScore: {
       type: Number,
       optional: true,
-      defaultValue: 0,
+      nullable: false,
       canRead: customBaseScoreReadAccess || ['guests'],
-      onInsert: (document: DbInsertion<T>): number => {
-        // default to 0 if empty
-        return document.baseScore || 0;
-      }
+      ...schemaDefaultValue(0),
```

Comment:
We should double-check whether this substitution is safe, since it adds `onUpdate` as well as `onCreate` (and a couple other fields). If it is, we don't need to explicitly specify `nullable: false` above